app/views/applications/show.html.erb

```diff
@@ -18,8 +19,12 @@
 <% end %>
 </section>
 <% @searched_pets.each do |pet| %>
-  <%=  pet.name %>
+  <%= pet.name %><%= form_with url: "/applications/#{@applicant.id}?pet=#{pet.id}", method: :patch, data: { turbo: false } do |form| %>  
+
+    <%= form.submit "Adopt this pet?" %>
+  <% end %>
+  
 <% end %>
-<h4>In Progress</h4>
 
 
+<h4>In Progress</h4>
```

Comment:
I did a little bit of research into this by the way, I think we can add a default value with a new column to our PetsApplications model with a migration
`add_column :status, :string, :default => "In Progress"